### PR TITLE
Update images

### PR DESCRIPTION
**Pull request by yuval-hazaz**

### CODE REVIEW

File: packages/amplication-client/src/Application/ApplicationLayout.tsx [hunk 1]
```diff
@@ -162,7 +162,6 @@ function ApplicationLayout({ match }: Props) {
           <Switch>
             <Route exact path="/:application/" component={ApplicationHome} />
             <Route
-              exact
```
Comment (iddan) [inline packages/amplication-client/src/Application/ApplicationLayout.tsx:165]: Not sure how does this relate to the PR

File: packages/amplication-client/src/VersionControl/PendingChanges.tsx [hunk 2]
```diff
@@ -36,6 +38,16 @@ type Props = {
 const PendingChanges = ({ match }: Props) => {
   const { application } = match.params;
   useBreadcrumbs(match.url, "Pending Changes");
+  const history = useHistory();
+
+  const stateMatch = useRouteMatch<{ state: string }>(
```
Comment (iddan) [inline packages/amplication-client/src/VersionControl/PendingChanges.tsx:43]: Rename to `match`

File: packages/amplication-client/src/VersionControl/PendingChanges.tsx [hunk 3]
```diff
@@ -36,6 +38,16 @@ type Props = {
...
+    "/:application/pending-changes/:state"
+  );
+
+  let state: string | null = null;
+  if (stateMatch) {
+    state = stateMatch.params.state;
+  }
```
Comment (iddan) [inline packages/amplication-client/src/VersionControl/PendingChanges.tsx:50]:
```suggestion
  const state = stateMatch ? stateMatch.params.state : null;
```

File: packages/amplication-client/src/VersionControl/PendingChanges.tsx [hunk 4]
```diff
@@ -65,6 +77,18 @@ const PendingChanges = ({ match }: Props) => {
     setDiscardDialogOpen(!discardDialogOpen);
   }, [discardDialogOpen, setDiscardDialogOpen]);
 
+  const handleCommitDialogCompleted = useCallback(() => {
```
Comment (iddan) [inline packages/amplication-client/src/VersionControl/PendingChanges.tsx:80]:
```suggestion
  const handleCommitDialogComplete = useCallback(() => {
```

File: packages/amplication-client/src/VersionControl/PendingChanges.tsx [hunk 5]
```diff
@@ -24,6 +25,7 @@ import useBreadcrumbs from "../Layout/use-breadcrumbs";
 
 const CLASS_NAME = "pending-changes";
 const POLL_INTERVAL = 2000;
+const STATE_COMMIT_DONE = "done";
```
Comment (iddan) [inline packages/amplication-client/src/VersionControl/PendingChanges.tsx:28]:
```suggestion
const COMMIT_STATE_DONE = "done";
```

Comment (iddan) [reply packages/amplication-client/src/VersionControl/PendingChanges.tsx:28]: Can you use a value from an enum from the server?

**Review by iddan — APPROVED** [review]

**Comment by iddan** [comment]
@yuval-hazaz please issue my comments in another PR

File: packages/amplication-client/src/VersionControl/PendingChanges.tsx (hunk 5 above)
Comment (yuval-hazaz) [reply packages/amplication-client/src/VersionControl/PendingChanges.tsx:28]: it has nothing to do with the server, It is literally a "client side" state that only describe the state of the page in order to decide what image to show when the list is empty

File: packages/amplication-client/src/Application/ApplicationLayout.tsx (hunk 1 above)
Comment (yuval-hazaz) [reply packages/amplication-client/src/Application/ApplicationLayout.tsx:165]: See below

**Pull request merged**